api/authenticator.py

```diff
@@ -2632,14 +2632,6 @@ def remote_exchange_authorization_code_for_access_token(self, _db, code):
         """
         raise NotImplementedError()
 
-    def remote_patron_lookup(self, access_token):
```

Comment:
On [line 2537](https://github.com/ThePalaceProject/circulation/pull/652/files/bc1393b235d9e2c1b7d8ab3e447672d8a581ea2a#diff-e4f8733d198e59b2bd6dc9532b19cb6387b9dfbe182f32474600bdbe9dc0ccbbR2537) there is method with same name and signature that is actually implemented :D though it only returns `None`

Reply:
Yes, you are correct. I thought it should be the one that is implemented, but it should be in fact this one since that would be used if this method is called. Thanks 👍